frontend/src/game/scenes/Collect.ts

```diff
@@ -106,8 +106,8 @@ export class Collect extends Scene {
     const screenCenterY = this.cameras.main.worldView.y + this.cameras.main.height / 2;
 
     return {
-      xPos: screenCenterX + (-150 + Math.random() * 300),
-      yPos: screenCenterY + (-200 + Math.random() * 400)
+      xPos: screenCenterX + (-500 + Math.random() * 1000),
+      yPos: screenCenterY + (-1 + Math.random() * 1)
```

Comment:
Multiplying by 1 isn't going to do anything. You could remove it.

Or you could change the value to a slightly higher number to see if it removes those awkward rows that were present in your screenshot. By smoothing out the starting height it might make it look more random. Maybe?

Reply:
Issue has been resolved.